### PR TITLE
Spring boot oci

### PR DESCRIPTION
**Pull request by nschnepf**
First pass review/edit for migration

### CODE REVIEW

File: install-spring-boot-on-an-oracle-instance.md [hunk 1]
```diff
@@ -22,19 +22,19 @@ redirect: https://developer.oracle.com/tutorials/install-spring-boot-on-an-oracl
...
 
 Key tasks include how to:
 
 * Set up a compartment for your development work.
-* Install your Oracle Linux instance and connect it to your Virtual Cloud Network (VCN). 
-    * Set up an Oracle Cloud Infrastructure virtual cloud network and related network services required for your host to connect to the internet.
-    * Set up `ssh` encryption keys to access your Oracle Linux Server.
+* Install your Oracle Linux instance and connect it to your Virtual Cloud Network (VCN).
+  * Set up an Oracle Cloud Infrastructure virtual cloud network and related network services required for your host to connect to the internet.
+  * Set up `ssh` encryption keys to access your Oracle Linux Server.
```
Comment (ttscoff) [inline install-spring-boot-on-an-oracle-instance.md:32]: List items should be indented 4 characters. This is for compatibility with ALL flavors of Markdown, even though 2 would work in most cases. 4 will ALWAYS work.

File: install-spring-boot-on-an-oracle-instance.md [hunk 2]
```diff
@@ -52,22 +52,22 @@ To successfully complete this tutorial, you must have the following:
...
-2. Open the navigation menu and click **Identity & Security**. Under **Identity**, click **Compartments**.
-3. Click **Create Compartment**.
+2. Open the navigation menu and select **Identity & Security**. Under **Identity**, select **Compartments**.
+3. Select **Create Compartment**.
 4. Fill in the following information:
     * **Name:** `<your-compartment-name>`
     * **Description:** `Compartment for <your-description>.`
     * **Parent Compartment:** `<your-tenancy>(root)`
-5. Click **Create Compartment**.
+5. Select **Create Compartment**.
```
Comment (ttscoff) [inline install-spring-boot-on-an-oracle-instance.md:70]: If this isn't already in our style guide feel free to contribute an addition. We see a lot of "click," "press," "choose," and "select" and coming up with a nomenclature we can stick with would be a good idea.

File: install-spring-boot-on-an-oracle-instance.md [hunk 3]
```diff
@@ -192,151 +193,161 @@ Follow these steps to select your VCN's public subnet and add the ingress rule.
...
 
     Now HTTP connections are allowed. Your VCN is configured for Spring Boot.
 
     You have successfully created an ingress rule that makes your instance available from the internet.
 
-## Install and Configure Spring Boot
+## Install and configure Spring Boot
 
-Next, install all the software needed for your Spring Boot application.
+Before the Spring Boot application is ready to use, you first configure the instance you created previously and then install 3 software packages, Git, OpenJDK 8, and Maven 3.6.
```
Comment (ttscoff) [inline install-spring-boot-on-an-oracle-instance.md:204]: (suggest a colon): 3 software packages: Git, Open...

File: install-spring-boot-on-an-oracle-instance.md [hunk 4]
```diff
@@ -364,40 +375,41 @@ Follow these steps to set up your Spring Boot application:
...
-        $ curl -X GET http://localhost:8080
+        curl -X GET http://localhost:8080
         ```
 
-    - From your browser connect to the public IP address assigned to your instance: **http://\<x.x.x.x>:8080**.
+    * From your browser, connect to the public IP address assigned to your instance: `http://<x.x.x.x>:8080`.
 
-    On your instance or in your browser, you see **Spring Boot Hello World!**
+    On either your instance or in your browser, you see
+    >**Spring Boot Hello World!**
```
Comment (ttscoff) [inline install-spring-boot-on-an-oracle-instance.md:406]: Add a space after the `>` (> **Spring)

**Review by ttscoff — CHANGES_REQUESTED** [review]
Great changes overall. Please see notes and make a final commit before I merge.

File: install-spring-boot-on-an-oracle-instance.md (hunk 2 above)
Comment (nschnepf) [reply install-spring-boot-on-an-oracle-instance.md:70]: I've gravitated towards something akin to 'select' in the past just so it's a little more general and can include input from mobile devices.

File: install-spring-boot-on-an-oracle-instance.md (hunk 1 above)
Comment (nschnepf) [reply install-spring-boot-on-an-oracle-instance.md:32]: Interesting. The linter didn't flag that as an issue so I didn't change the existing list formatting. But I'll definitely make sure to check that in the future!

Comment (nschnepf) [reply install-spring-boot-on-an-oracle-instance.md:32]: Updated

File: install-spring-boot-on-an-oracle-instance.md (hunk 3 above)
Comment (nschnepf) [reply install-spring-boot-on-an-oracle-instance.md:204]: Included

File: install-spring-boot-on-an-oracle-instance.md (hunk 4 above)
Comment (nschnepf) [reply install-spring-boot-on-an-oracle-instance.md:406]: Added

**Review by ttscoff — APPROVED** [review]

**Pull request merged**